IPython/__init__.py

```diff
@@ -47,9 +47,7 @@
 from .core.application import Application
 from .terminal.embed import embed
 
-from .core.error import TryNext
 from .core.interactiveshell import InteractiveShell
-from .testing import test
```

Comment:
Things imported into `__init__` are usually deliberate in order to expose them in the public API. Static analysis tools should really ignore unused imports in `__init__`.

I'm OK with dropping TryNext from the top level API, but I'd like to keep `test()` - I think the idea was to have a convention that the Scipy stack projects could all be tested by doing `import project; project.test()`.